cmake/modules/AddSwift.cmake

```diff
@@ -223,7 +223,6 @@ function(_add_variant_swift_compile_flags
   list(APPEND result
       "-sdk" "${SWIFT_SDK_${sdk}_PATH}"
       "-target" "${SWIFT_SDK_${sdk}_ARCH_${arch}_TRIPLE}"
-      "-resource-dir" "${SWIFTLIB_DIR}")
```

Comment:
It is actually necessary (because the compiler probably isn't in the build directory), but it could be split out separately and committed *first* if it really mattered. I don't think it really matters, though.